### PR TITLE
[14.0][FIX] sale_timesheet_order_line_no_update: dumps domain

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Review by HekkiMelody — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@pedrobaeza quick bugfix + maintainer :)

File: sale_timesheet_order_line_no_update/models/project.py [hunk 1]
```diff
@@ -33,15 +35,17 @@ def _compute_sale_line_id_domain(self):
...
-            rec.sale_line_id_domain = [
-                ("is_service", "=", True),
-                ("is_expense", "=", False),
-                (field, "=", value),
-                ("state", "in", ["sale", "done"]),
-                "|",
-                ("company_id", "=", False),
-                ("company_id", "=", rec.company_id.id),
-            ]
+            rec.sale_line_id_domain = json.dumps(
```
Comment (pedrobaeza) [inline sale_timesheet_order_line_no_update/models/project.py]: For this to work, the field should be binary, not char AFAIK.

Comment (ilyasProgrammer) [reply sale_timesheet_order_line_no_update/models/project.py]: According to the docs it is Char: https://github.com/OCA/web/blob/14.0/web_domain_field/readme/USAGE.rst

Comment (francesco-ooops) [reply sale_timesheet_order_line_no_update/models/project.py]: @pedrobaeza shall we go?

Comment (pedrobaeza) [reply sale_timesheet_order_line_no_update/models/project.py]: This can be done without requiring such module with a binary field.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Comment by francesco-ooops** [comment]
@pedrobaeza try again?

**Pull request merged**